### ISSUE TITLE
Carthage build failed

### ISSUE THREAD

**Issue by nezhyborets** [Shopify/mobile-buy-sdk-ios#279]
I'm trying to get version 2.0.0 from Master with Carthage.

```shell
carthage update
*** Fetching mobile-buy-sdk-ios
*** Checking out mobile-buy-sdk-ios at "2.0.0"
*** xcodebuild output can be found in /var/folders/5q/l8ylk7zn7bdb1hllg10b36fw0000gn/T/carthage-xcodebuild.4pxocK.log
*** Building scheme "Buy" in Mobile Buy SDK.xcodeproj
** BUILD FAILED **


The following build commands failed:
	CompileC /Users/oleksiinezhyborets/Library/Developer/Xcode/DerivedData/Mobile_Buy_SDK-bgizupzabnviyxczrhaxsmqdhsxo/Build/Intermediates/Mobile\ Buy\ SDK.build/Release-iphoneos/Buy.build/Objects-normal/armv7/BUYClient+Storefront.o Mobile\ Buy\ SDK/Data/BUYClient+Storefront.m normal armv7 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
/Users/oleksiinezhyborets/Projects/KidsSizingApp/FutureShoes-RT-SVN/Carthage/Checkouts/mobile-buy-sdk-ios/Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient+Storefront.m:140:48: error: implicit conversion loses integer precision: 'const NSUInteger' (aka 'const unsigned int') to 'BOOL' (aka 'signed char') [-Werror,-Wconversion]
A shell task (/usr/bin/xcrun xcodebuild -project /Users/oleksiinezhyborets/Projects/KidsSizingApp/FutureShoes-RT-SVN/Carthage/Checkouts/mobile-buy-sdk-ios/Mobile Buy SDK/Mobile Buy SDK.xcodeproj -scheme Buy -configuration Release -sdk iphoneos ONLY_ACTIVE_ARCH=NO BITCODE_GENERATION_MODE=bitcode CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES clean build) failed with exit code 65:
** BUILD FAILED **


The following build commands failed:
	CompileC /Users/oleksiinezhyborets/Library/Developer/Xcode/DerivedData/Mobile_Buy_SDK-bgizupzabnviyxczrhaxsmqdhsxo/Build/Intermediates/Mobile\ Buy\ SDK.build/Release-iphoneos/Buy.build/Objects-normal/armv7/BUYClient+Storefront.o Mobile\ Buy\ SDK/Data/BUYClient+Storefront.m normal armv7 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
```

**Comment by bgulanowski** [comment]
That should be fixed by this:

https://github.com/Shopify/mobile-buy-sdk-ios/pull/280

**Comment by davidmuzi** [comment]
Fixed by #280
